test/system/sign_in_test.rb

```diff
@@ -0,0 +1,14 @@
+require "application_system_test_case"
+
+class SignInTest < ApplicationSystemTestCase
```

Comment:
_nitpick_ _suggestion_

As this test is testing for creating a new user, I think you should rename it to: 

```suggestion
class RegistrationTest < ApplicationSystemTestCase
```

Along with changing the file name accordingly to `test/system/registration_test.rb`